docs/proposals/koordlet/20230728-support-cold-memory-compute.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Support to collect and report cold memory
+authors:
+  - "@BUPT-wxq"
+reviewers:
+  - "@saintube"
+  - "@zwzhang0107"
+  - "@eahydra"
+  - "@hormes"
+  - "@FillZpp"
+creation-date: 2023-07-28
+last-updated: 2023-07-28
+---
+
+## Table of Contents
+
+
+- [Title](#title)
+  - [Table of Contents](#table-of-contents)
+  - [Glossary](#glossary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals/Future Work](#non-goalsfuture-work)
+  - [Proposal](#proposal)
+    - [Design](#design)
+    - [Collect idle memory info and insert memory with hot page](#collect-idle-memory-info-and-insert-memory-with-hot-page)
+	- [Report memory usage including hot page](#report-memory-usage-including-hot-page)
+	- [Define cold memory API](#define-cold-memory-api)
+
+## Glossary
+
+**cold page**
+
+A cold page is a page that keeps idle for a long time in the page cache.
+
+**hot page**
+
+A cold page is a page that is often used in the page cache.
+
+**kidled**
+
+Kided, an open source cold memory collection solution, identifies the hot and cold conditions of nodes, pod and container memory in the cluster. 
+
+## Summary
+
+To support more fine-grained memory overselling, incorporate cold memory compute into memory usage in koordinator.
+
+## Motivation
+
+Currently, the memory usage of nodes and pods collected by koordlet does not contain page cache. It is too aggressive. There are many hot pages that should be counted as memory usage in page cache .That will also help identify cold page and more fine-grained oversell memory. This proposal focuses on cold memory collection and report. 
```

Comment:
We'd better add some examples.
For instance, which case the page cache can help? Why the kernel does not proactively reclaim the page cache? Why the cold pages are suitable to reclaim but hot pages not?